### ISSUE TITLE
Rebuild this project PlugRest

### ISSUE THREAD

**Issue by rejon** [freebassel/api#3]
Should give us both the website documentation, and initial simple output, but could be a good testbed of pushing the resource delivery system of PlugRest to the max:

https://github.com/christopheradams/plug_rest

This would unlock:

- [ ] adding svg output
- [ ] json output
- [ ] mp3 output
- [ ] midi output

hahaha, make a long list of resources this DAYS project can output and go wild.

Actually, probably better to just make a brand new REPO called days.freebassel.org and move back to idea that website + api are integrated.

**Comment by rejon** [comment]
@christopheradams @mlinksva

**Comment by rejon** [comment]
@clemsos

**Comment by christopheradams** [comment]
I was thinking about a more general service called **DaysSince** where you can create a resource with a particular date and then track the "days since", and show a message and link. That would replace this API, but allow the same deal for more events. That is very simple.

But I also get what you're saying: have a new resource for each day (so more like the DAYS project) where we have to fill in each day with different media representations.

**Comment by rejon** [comment]
Yeah that would also work.

I would just like to keep days and disappeared but yeah there are artworks created for many of the days since he was detained that could be served if we have a method for asking for art or other info made for say day 1 of bassel being detained. Many people made things for these days.

**Comment by mlinksva** [comment]
Is there any catalog of freebassel art? If not I suppose starting with a very simple json/yaml/whatever file with entries including date could be used to serve methods imagined above.

**Comment by rejon** [comment]
Good idea. We have currently in spreadsheet google doc. Christopher what format would be needed for this?

Jon

**Comment by christopheradams** [comment]
Whatever format you want to be able to edit in. If the data is simple enough and you want it public, a YAML file in a Github repo would work.
